src/pages/AppDetail.svelte

```diff
@@ -1,4 +1,5 @@
 <script>
+  import Footer from "../components/Footer.svelte";
```

Comment:
eslint requires you to add a new line after the import.

![Screenshot 2020-10-11 at 7 49 38 PM](https://user-images.githubusercontent.com/19612755/95681744-072cb780-0bfb-11eb-9908-514dcae79aad.png)

Add a new line after the import and you are good to go.

Reply:
Hey, thank you for showing the error. However, where did you find this error? I mean did you click the circle ci icon? I could reach here. Build still seems to fail after adding a new line after import too.

Reply:
Yes, I checked why the build failed by clicking the icon. 

I reproduced this linting error locally by removing the new line after the imports.

You can read about this issue more here
[https://github.com/sveltejs/eslint-plugin-svelte3/issues/41](https://github.com/sveltejs/eslint-plugin-svelte3/issues/41)

It looks like it is a formatting issue,
![Screenshot 2020-10-11 at 9 45 59 PM](https://user-images.githubusercontent.com/19612755/95684439-3ba86f80-0c0b-11eb-8b16-60166af8ae72.png)


Try it with  ```npm run format```